slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/dao/ValidatorsDaoTest.java

```diff
@@ -81,8 +86,50 @@ public void storesUnregisteredValidatorsInDb(final Handle handle) {
     assertThat(validators.get(4)).isEqualToComparingFieldByField(new Validator(5, Bytes.of(104)));
   }
 
-  private void insertValidator(final Handle h, final int i) {
-    final byte[] value = Bytes.of(i).toArrayUnsafe();
-    h.execute("INSERT INTO validators (public_key) VALUES (?)", value);
+  @Test
+  public void isEnabledReturnsTrueForEnabledValidator(final Handle handle) {
+    insertValidator(handle, Bytes.of(1), true);
+    assertThat(new ValidatorsDao().isEnabled(handle, 1)).isTrue();
+  }
+
+  @Test
+  public void isEnabledReturnsTrueForExistingValidator() {
+    final TestDatabaseInfo testDatabaseInfo = DatabaseUtil.createWithoutMigration();
+    final Jdbi jdbi = testDatabaseInfo.getJdbi();
+
+    try (Handle handle = jdbi.open()) {
+      final String versionBeforeEnableFlag = String.valueOf(VALIDATOR_ENABLE_FLAG_VERSION - 1);
```

Comment:
should we declare constant '9' as that is known to be not have validator enabled logic? Otherwise, when version is 11, the above would return 10.

Reply:
or VALIDATOR_ENABLE_FLAG_VERSION is meant to remain 10 and not to be incremented?

Reply:
Was going to leave VALIDATOR_ENABLE_FLAG_VERSION as a constant since that the version the enable flag is added in